src/components/price/Price.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Image, PriceContent, PriceWrapper, Text, Title, WorkButton } from './styledPrice'
+import { PriceData } from '../../constants/priceConst'
+const { title, text, button } = PriceData;
+
+const Price = () => {
+  return (
+    <PriceWrapper>
+      <PriceContent>
+        <Title>{title}</Title>
+        <Image />
```

Comment:
I see what you mean about the misspelling in the image.  I would leave it and perhaps we can reach out to the designer and some point and get an updated image.  Seems easier for him to make an adjustment then redo your whole code structure here